src/main/java/mate/jdbc/service/impl/ManufacturerServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package mate.jdbc.service.impl;
+
+import java.util.List;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.ManufacturerService;
+
+@Service
+public class ManufacturerServiceImpl implements ManufacturerService {
+    @Inject
+    private ManufacturerDao manufacturerDao;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        return manufacturerDao.create(manufacturer);
+    }
+
+    @Override
+    public Manufacturer get(Long id) {
+        return manufacturerDao.get(id).get();
```

Comment:
Make here informative exception message

Reply:
![image](https://user-images.githubusercontent.com/98731880/173599945-193d9567-6f00-4378-9ce8-78b2d46aa982.png)
If I understand correctly, it says to use get() ?

Reply:
Yeah, you're right🧐